### PR TITLE
Add Expat submodule

### PR DESCRIPTION
**Pull request by vadz**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -5,3 +5,6 @@
 	path = src/zlib
 	url = https://github.com/wxWidgets/zlib.git
 	branch = wx
+[submodule "src/expat"]
+	path = src/expat
+	url = https://github.com/wxWidgets/libexpat.git
```
Comment (MaartenBent) [inline .gitmodules]: Add `branch = wx` here as well?

Comment (vadz) [reply .gitmodules]: Yes, thanks. I'll update this PR soon, once the CI builds for the png one finish and I can merge that one as they're going to conflict with each other.

File: .gitmodules [hunk 2]
```diff
@@ -9,3 +9,7 @@
 	path = src/png
 	url = https://github.com/wxWidgets/libpng.git
 	branch = wx
+[submodule "src/expat"]
+	path = src/expat
+	url = https://github.com/wxWidgets/libexpat.git
+    branch = wx
```
Comment (MaartenBent) [inline .gitmodules]: spaces -> tab

Comment (vadz) [reply .gitmodules]: Fixed, thanks for noticing.

**Comment by vadz** [comment]
Merged as 2cee986e03d68e91071056e3e7f728dbd1d442a9

**Pull request closed without merging**